README.md

```diff
@@ -275,8 +275,8 @@ It is possible to use travis-ci in order to build and upload your tagged release
 	```
 1. Run `travis setup releases`, follow documentation [here](https://docs.travis-ci.com/user/deployment/releases/)
 
-	This command will encode your github credentials into the .travis.yml file in order to let travis upload the release to github.com
-	When prompted for the file to upload, enter $FRAMEWORK_NAME.framework.zip
+	This command will encode your Github credentials into the `.travis.yml` file in order to let travis upload the release to Github.com
```

Comment:
This should really be `GitHub` with a capital _H_. Would you mind fixing that minor detail?